cmd/tarmak/cmd/cluster.go

```diff
@@ -95,6 +95,25 @@ func clusterFlagDryRun(fs *flag.FlagSet, store *bool) {
 	)
 }
 
+func clusterKubeconfigFlags(fs *flag.FlagSet) {
+	store := &globalFlags.Cluster.Kubeconfig
+
+	fs.BoolVar(
+		&store.PublicAPIEndpoint,
+		"public-api-endpoint",
+		false,
```

Comment:
This should be true for clusters where there is a public API endpoint available, false if not

Both values should be able to be overriden